README.md

```
@@ -1,91 +1,124 @@
 
-| 기존 서비스 | 문제점과 한계 | 우리 서비스 |
-|-------------|---------------|-------------|
-| 온라인 모각코 | 진입장벽 존재 (접속 규칙, 화면/캠 공유) | 규칙 없이 즉시 참여 가능 |
-| 메타버스 (ZEP, 게더타운) | 개발자 특화 기능 부재 | GitHub 연동으로 개발자 특화 기능 제공 |
-| 디스코드/슬랙 | 공간감, 시각적 성취 부족 | 함께한다는 유대감, 시각적 성취 제공 |
-| 오프라인 모각코 | 지리적 한계, 내향적인 성향 | 지리적 제한 없이 온라인으로 가볍게 접속 |
+### 1️⃣ 함께 성장하는 가상 세계
+모두의 활동이 모여 맵이 변화. 7일 시즌제로 주기적 리셋이 이루어지고, 테마가 변경됨.
 
----
+### 1️⃣ 함께 성장하는 가상 세계
```

markdownlint-cli2 (0.20.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)